package.json

```diff
@@ -31,49 +31,58 @@
   },
   "repository": "git@github.com:html-next/ember-gestures.git",
   "scripts": {
-    "start": "ember server",
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "start": "ember serve",
+    "test": "ember test",
     "build": "ember build",
-    "test": "ember try:each"
+    "test:all": "ember try:each"
   },
   "homepage": "https://github.com/html-next/ember-gestures",
   "bugs": "https://github.com/html-next/ember-gestures/issues",
   "dependencies": {
+    "animation-frame": "~0.2.4",
     "broccoli-funnel": "^2.0.1",
     "broccoli-merge-trees": "^2.0.0",
     "broccoli-stew": "^1.5.0",
     "ember-cli-babel": "^6.8.2",
     "ember-cli-htmlbars": "^2.0.1",
-    "ember-cli-version-checker": "^1.2.0",
-    "ember-getowner-polyfill": "^1.2.3",
+    "ember-cli-version-checker": "^2.1.0",
+    "ember-getowner-polyfill": "^2.2.0",
     "hammerjs": "^2.0.8",
+    "marked": "~0.3.5",
```

Comment:
where is this used? it's a lot of code? (brought over from bower)

Reply:
moved app.import to ember-cli-build, shouldn't bloat size of code shipped to consumers